src/MentoringAndCoaching/controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from "express";
+import { searchMentorShipService } from "./services";
+
+export const search = async (req: Request, res: Response) => {
+  const { data, status = 200 } = await searchMentorShipService(req.body);
+
+  return res.status(status).send({ data });
```

Comment:
Please remove the data key in send call. We are sending data directly.

Reply:
done